eZ/Publish/Core/Persistence/Cache/UrlAliasHandler.php

```diff
@@ -205,7 +204,8 @@ public function lookup($url)
             try {
                 $this->logger->logCall(__METHOD__, array('url' => $url));
                 $urlAlias = $this->persistenceHandler->urlAliasHandler()->lookup($url);
-                $cache->set($urlAlias->id);
+                $urlAliasId = $urlAlias->id;
```

Comment:
Why the temp variable?

Also: you can warmup the cache here, but make that change a separate commit please, as it is a different topic, if you want, rebase PR and make that the first commit _(remember to adapt tests if needed for the commit)_.